packages/amplify_datastore_plugin_interface/lib/amplify_datastore_plugin_interface.dart

```diff
@@ -60,12 +64,13 @@ abstract class DataStorePluginInterface extends AmplifyPluginInterface {
     this.syncInterval,
     this.syncMaxRecords,
     this.syncPageSize,
+    this.authModeStrategy = AuthModeStrategy.default$,
   }) : super(token: token);
 
   /// Internal use constructor
   @protected
   DataStorePluginInterface.tokenOnly({required Object token})
-      : super(token: token);
+      : this(token: token, modelProvider: null);
```

Comment:
What's the reason changing this line?

Reply:
It's identical to before, just saves me repeating myself again. Alternative is:

```dart
  DataStorePluginInterface.tokenOnly({required Object token})
      : authModeStrategy = AuthModeStrategy.default$,
        super(token: token);
```